tests/cli/bootsys/test_bos.py

```diff
@@ -524,5 +573,39 @@ def test_do_bos_shutdowns_without_prompt(self):
         self.mock_do_bos_ops.assert_called_once_with('shutdown', 60, limit=self.limit, recursive=False)
 
 
+class TestDoBosReboots(ExtendedTestCase):
+    """Tests for the do_bos_reboots() function."""
+
+    def setUp(self):
+        self.mock_prompt = patch('sat.cli.bootsys.bos.prompt_continue').start()
+        self.mock_do_bos_ops = patch(
+            'sat.cli.bootsys.bos.do_bos_operations').start()
+        self.mock_get_config = patch(
+            'sat.cli.bootsys.bos.get_config_value', return_value=60).start()
```

Comment:
If you change the timeout logic, I would suggest changing this to something like:

```suggestion
        self.mock_get_config = patch(
            'sat.cli.bootsys.bos.get_config_value', side_effect=[60, 90]).start()
```

And then assert that the `mock_do_bos_ops` is called with a timeout of `150` to distinguish whether it's adding the two timeouts or doubling them.